site/specs-source/multiFactorAuth.json

```
@@ -227,8 +227,8 @@
     },
     "/accounts/{accountId}/code/verify": {
       "post": {
-        "summary": "Verify a previously sent two-factor authentication code",
-        "description": "Allows a user to verify an MFA code",
+        "summary": "Two-Factor Verify Code",
```

```suggestion
        "summary": "Verify Authentication Code",
```
I know we had the prior in the old docs but... Thoughts?